config.yaml

```diff
@@ -19,3 +19,9 @@ options:
       The public-facing base URL that clients use to access this Homeserver.
       Defaults to https://<server_name>/. Only used if there is integration with
       SAML integrator charm.
+  enable_mjolnir:
```

Comment:
Thinking about our use case, would it make sense to have it enabled by default?

Reply:
It's a good suggestion but I would say no because Mjolnir is a tool that can be run externally. So, enabling this one inside the charm might cause conflicts.